api/graphite.go

```diff
@@ -673,3 +673,143 @@ func getLocation(desc string) (*time.Location, error) {
 	}
 	return time.LoadLocation(desc)
 }
+
+func (s *Server) graphiteTagDetails(ctx *middleware.Context, request models.GraphiteTagDetails) {
+	tag := ctx.Params(":tag")
+	if len(tag) <= 0 {
+		response.Write(ctx, response.NewError(http.StatusBadRequest, "not tag specified"))
```

Comment:
another benefit of specifying tag as a field is that we get validation and don't need to validate in our handlers anymore.

Reply:
for this one we still need to do that, because that's the url schema that graphite expects. but yes, for the cluster-internal call on `/index` we can omit that